docs/answers.md

```diff
@@ -38,6 +38,58 @@ const annotateData = (array) => array.reduce(reducer, [])
 const newData = annotateData(data)
 ```
 
+## Objects and Classes
+
+### Delays
+
+```{js}
+class Delay {
+  constructor(initialValue){
```

Comment:
Please use a space between the method name and the opening parenthesis, and between the closing parenthesis and curly brace:
```
constructor (initialValue) {
```

Reply:
Thanks for the review and have acknowledged the required changes to be made 👍